4.3/README.md

```diff
@@ -214,8 +307,15 @@ In addition, we also provide a basic `contrib/docker-compose.yml` template which
 Besides the basic docker-compose.yml, there are also some advanced YAML configurations available in the `contrib` folder.
 These files follow the naming convention of `docker-compose-*.yml` and contain comments about the specific use case.
 
+## Nominatim UI
+
+You have to set the permissions for the Webserver group(default www-data) at /nominatim/website to 755.
```

Comment:
I'm sorry, this has absolutely nothing do to with this container image and should be part of the nominatim-ui docs.

If you want you can create a separate discussion where you document how to set up the UI and then add a link to the "Assorted use cases section".

Reply:
Removed the Permission doku and made a PR there